src/Compilers/CSharp/Test/Emit3/Semantics/PrimaryConstructorTests.cs

```diff
@@ -3133,7 +3133,12 @@ public class C : System.Attribute
     int M1() => P1;
 }
 ";
-            var comp = CreateCompilation(source);
+            var comp = CreateCompilation(source, parseOptions: TestOptions.RegularDefault.WithFeature("run-nullable-analysis", "never"));
+            var test1 = comp.GetTypeByMetadataName("Test1");
+            var ctor = test1.InstanceConstructors.Where(c => !c.IsDefaultValueTypeConstructor()).Single();
+
+            ctor.GetAttributes();
```

Comment:
Consider asserting at least number of the attributes returned by this call.

Reply:
> Consider asserting at least number of the attributes returned by this call.

This is not interesting. We are requesting the attributes to get symbol to certain state.